references/cohort_consensus.fasta

```diff
@@ -0,0 +1 @@
+NC_045512.2.fasta
```

Comment:
Given diffs as this one in the pull-request, I think you forgot to rebase against the latest verison of `sars-cov2-dev`.